src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/showConfirmation.js

```diff
@@ -118,6 +118,7 @@ function showConfirmation(req, res, next) {
 
       // making sure Adyen_paymentMethod is populated before calling clearAdyenData()
```

Comment:
there's a comment about clearing data but the command to do so wasn't there? that is strange

Reply:
Yeah it's weird.. I'm going to delete this comment anyways. Looks like a dev thinking out loud rather than a helpful comment